dev/planning-example.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# ============================
+# Observation Planning Toolbox
+# ============================
+"""
+The observation planning toolbox consists of a number of entities that
+simplify the planning and scheduling of observations by abstracting much
+of the boilerplate code necessary to accomplish the same thing using
+core astropy or ephemerides objects.  These can be thought of as common
+idioms or convenience classes and functions that would be recreated over
+and over by astronomers and or software engineers building telescope
+scheduling software.
+
+Important assumptions:
+
+- It should be convenient to specify dates and times in the local timezone
+  of the observer.
+
+- It should be convenient to do timezone conversions easily.
+
+  This means methods that can take dates can either take astropy
+  dates (as used internally) or python datetime objects (not naive, but
+  tagged with timezone).  So dates can be passed in as timezone-aware
+  datetime objects and results can be easily converted between such.
```

Comment:
Good news on this front: it looks to me like `astropy.time.Time` does everything exactly like you'd expect with timezones.  E.g. ``Time(datetimeobjwithtzinfo)`` transparently converts to utc, so all that needs to be done with accepting times is ``Time(input)`` and that will work fine if `input` is a `Time` or a `datetime`.  And the ``datetime`` output of `Time` is always UTC, so anywhere that you want to get back a local time it can internally just be ``timeobj.datetime.astimezone(self.timezone)``